internal/ec2provider/compute.go

```diff
@@ -34,22 +34,21 @@ type ComputeManager interface {
 }
 
 type computeManager struct {
-	region    string
-	ec2Client *awsutils.Client
+	clientTimeoutSeconds int
+	ec2Client            *awsutils.Client
 }
 
 func NewComputeManager(ctx context.Context) (*computeManager, error) {
 	cfg := config.Config()
-	region := cfg.Region
 
-	ec2, err := awsutils.NewEc2Client(region)
+	ec2, err := awsutils.NewEc2Client(cfg.HealthCheckIntervalSeconds)
```

Comment:
The `HealthcheckIntervalSeconds` shouldn't be used by an EC2 client.  It should only be used in code that is doing a periodic health check.

Reply:
This is asked by @SaranBalaji90 , to implement timeout for all clients. 
Ref : https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/custom-http/

Reply:
Right, but `HealthcheckIntervalSeconds` specifically is the interval in seconds between _Health Checks_ (not a timeout for EC2 client connections).  The timeout setting should be a _different_ parameter that isn't already used for some other behavior.

Reply:
got it, renamed the param